### ISSUE TITLE
[UI]Build the Card Component

### ISSUE THREAD

**Issue by varisha-025** [varisha-025/BizRank#15]
The design of brand cards is attatched to the issue.

![Image](https://user-images.githubusercontent.com/81171573/194770406-737d28be-5a41-44b8-aa5f-f5f45f6fbca2.jpeg)
![Image](https://user-images.githubusercontent.com/81171573/194770403-974db567-4bd7-454e-a10a-62a39d5491c3.jpeg)
![Image](https://user-images.githubusercontent.com/81171573/194770404-91cb8bb9-5628-47b4-b196-49f9544139f9.jpeg)
![Image](https://user-images.githubusercontent.com/81171573/194770405-c65d410c-c911-46f8-9762-10704805ac84.jpeg)
![Image](https://user-images.githubusercontent.com/81171573/194770402-93c66e80-c874-430b-8626-49adb1073aa3.jpeg)

**Comment by avezqureshi14** [comment]
can you assign this to me

**Comment by varisha-025** [comment]
@avezqureshi14 Will you be able to build it using material ui?

**Comment by avezqureshi14** [comment]
yeah you can checkout my project which i had build using React JS and Material UI [https://avezqureshi.netlify.app/](url)

**Comment by varisha-025** [comment]
Okay. I have assigned you the issue. How much time do you need for this issue?

**Comment by avezqureshi14** [comment]
I will try to make the PULL REQUEST till or before 15th of October

**Comment by varisha-025** [comment]
Try to do it before 13th October @avezqureshi14

**Comment by avezqureshi14** [comment]
OK done and thanks @varisha-025 for assigning me this issue

**Comment by varisha-025** [comment]
I didn't get you. But to be clear, treat this card component like a product, bunch of this card which will be rendered on the dashboard, search operations will be done on this card as well.

**Comment by avezqureshi14** [comment]
Ok @varisha-025 I got it, and I will try to make the PULL REQUEST before 13th of October as you said

**Comment by varisha-025** [comment]
@avezqureshi14 Have you completed building the component. I have already given you extra time.

**Comment by avezqureshi14** [comment]
Sorry for the Delay, I had Completed Building the Component, and Just now I made the Pull request

**Comment by varisha-025** [comment]
@avezqureshi14 . Actually I am travelling right now and will reach the destination tomorrow.

**Comment by avezqureshi14** [comment]
ok @varisha-025 dii take your time 😊

**Comment by varisha-025** [comment]
@KrunalGupta02 I have assigned this issue to you, not the 3rd UI design

**Comment by KrunalGupta02** [comment]
@varisha-025 actually i want to work on 3rd UI design this task will take time plz if possible kindly assign me 3rd UI design
